### PR TITLE
added thumbnail

### PR DESCRIPTION
**Pull request by YiJie-Zhu**

### CODE REVIEW

File: lib/presentation/engage/bible_series/widgets/image_body.dart [hunk 1]
```diff
@@ -164,58 +163,72 @@ class _ImageInputBodyState extends State<ImageInputBodyState> {
...
 }
+
+Widget imageWidget(CompletionsState state, String contentNum) {
+  if (state.thumbnailURL != null && state.thumbnailURL[contentNum] != null) {
+    return Image.network(
+      state.thumbnailURL[contentNum][0],
+      fit: BoxFit.fill,
+    );
+  } else if (state.localImage != null && state.localImage[contentNum] != null) {
+    return Image.file(state.localImage[contentNum][0]);
```
Comment (JoshuaRamsamooj) [inline lib/presentation/engage/bible_series/widgets/image_body.dart:230]: Do we need the BoxFit.fill here as well?

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/image_body.dart:230]: yeah it enlarges the images no matter its pixel resolution

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/image_body.dart:230]: so all the thumbnail and image would be the same size

Comment (JoshuaRamsamooj) [reply lib/presentation/engage/bible_series/widgets/image_body.dart:230]: So do we need it here? I noticed that we have BoxFill for the network image but not for the file image.

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/image_body.dart:230]: for file image, the image is larger than the container size, so it gets restricted. For thumbnails, it is smaller so a BoxFill is needed to stretch it to however large the container is

**Pull request merged**